data/remnant/remnant news.txt

```diff
@@ -213,6 +213,8 @@ news "remnant security advisory"
 news "remnant incomprehensible"
 	location
 		government "Remnant"
+	to show
+		not "remnant: sign studies complete"
```

Comment:
```suggestion
```

I think it would be better to have these be a mix. Instead of basically eliminating the incomprehensible and replace it with the somewhat comprehensible list, I think it is better to leave the existing Remnant news as-is. That way, the news will become a mix with sometimes the player does understand them, sometimes they don't.  We could then add additional comprehensible news set triggered at the end of the Cognizance missions. This would result in a gradient of slowly increasing amount of comprehensible news over time.

Reply:
Have to agree here, the player isn't completely fluent, so having news messages appear where they don't understand something makes sense.